src/sentry/models/projectownership.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import
+
+from jsonfield import JSONField
+
+from django.db import models
+from django.utils import timezone
+from sentry.db.models import Model, sane_repr
+from sentry.db.models.fields import FlexibleForeignKey
+
+
+class ProjectOwnership(Model):
+    __core__ = True
+
+    project = FlexibleForeignKey('sentry.Project', unique=True)
+    raw = models.TextField(null=True)
```

Comment:
This will be the raw, unformatted input that comes from the frontend and is what is digested from the API.